### PR TITLE
add ZECTEST, fix LTC/LTCTEST

### PR DESCRIPTION
**Pull request by saloppe73**

### CODE REVIEW

File: hdwallet/cryptocurrencies.py [hunk 1]
```diff
@@ -3213,18 +3213,18 @@ class LitecoinMainnet(Cryptocurrency):
...
         "P2WSH": None,
         "P2WSH_IN_P2SH": None
     })
     EXTENDED_PUBLIC_KEY = ExtendedPublicKey({
-        "P2PKH": 0x019da462,
-        "P2SH": 0x019da462,
-        "P2WPKH": 0x04b24746,
-        "P2WPKH_IN_P2SH": 0x01b26ef6,
+        "P2PKH": 0x488b21e,
+        "P2SH": 0x488b21e,
```
Comment (meherett) [inline hdwallet/cryptocurrencies.py:3225]: It works in both P2PKH and P2SH but the standard of Litecoin EPrivateKey is ``0x019d9cfe`` and  EPublicKey is ``0x019da462``.

**Review by meherett — CHANGES_REQUESTED** [review]

File: hdwallet/cryptocurrencies.py (hunk 1 above)
Comment (meherett) [reply hdwallet/cryptocurrencies.py:3225]: Use this on LitecoinMainnet

```python3
    EXTENDED_PRIVATE_KEY = ExtendedPrivateKey({
        "P2PKH": 0x019d9cfe,  # 0x488ade4
        "P2SH": 0x019d9cfe,  # 0x488ade4
        "P2WPKH": None,
        "P2WPKH_IN_P2SH": None,
        "P2WSH": None,
        "P2WSH_IN_P2SH": None
    })
    EXTENDED_PUBLIC_KEY = ExtendedPublicKey({
        "P2PKH": 0x019da462,  # 0x488b21e
        "P2SH": 0x019da462,  # 0x488b21e
        "P2WPKH": None,
        "P2WPKH_IN_P2SH": None,
        "P2WSH": None,
        "P2WSH_IN_P2SH": None
    })
```

Comment (saloppe73) [reply hdwallet/cryptocurrencies.py:3225]: Got the data by the ltc chainparams.
https://github.com/litecoin-project/litecoin/blob/master/src/chainparams.cpp#L132

Comment (meherett) [reply hdwallet/cryptocurrencies.py:3225]: Okay, Maybe I will include option to access both extends keys next time. Good Job :)

**Pull request merged**